fighthealthinsurance/common_view_logic.py

```
@@ -475,9 +480,14 @@ def choose_appeal(
         # Get the current info
-        denial: Denial = Denial.objects.filter(
-            denial_id=denial_id, hashed_email=hashed_email, semi_sekret=semi_sekret
-        ).get()
+        try:
+            denial: Denial = Denial.objects.filter(
+                denial_id=denial_id, hashed_email=hashed_email, semi_sekret=semi_sekret
+            ).get()
+        except Denial.DoesNotExist:
+            raise Exception(
+                f"Could not find denial {denial_id} for the provided email and secret."
+            )
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)